weblab/experiments/models.py

```diff
@@ -159,7 +159,7 @@ class Meta:
 
     @property
     def name(self):
-        return str(self.run_number)
+        return '{:%Y-%m-%d %H:%M:%S}'.format(self.created_at)
```

Comment:
We should probably have a shared constant somewhere that determines the output format for dates

Reply:
At present I think it's only used here and in the corresponding test. If it occurs more then yes.